http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/OpenshiftTLSRegistryIT.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.ts.http.restclient.reactive;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+
+@Tag("QUARKUS-5664")
+@QuarkusScenario
+@Disabled("https://github.com/quarkus-qe/quarkus-test-framework/issues/1052")
```

Comment:
Even if https://github.com/quarkus-qe/quarkus-test-framework/issues/1052 I'd expect these tests still to fail because hostname verification will fail (hostname will never be `localhost`). This would have a value if it worked after https://github.com/quarkus-qe/quarkus-test-framework/issues/1052 got fixed. If you want to keep it, you may want to warn that tests are expected to fail and suggestion how to fix it?